logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -206,7 +210,19 @@ class ConversationDataSource(
             }
     }
 
+    override suspend fun getOne2OneConversationDetailsByUserId(otherUserId: UserId): Either<StorageFailure, ConversationDetails.OneOne?> {
+        return wrapStorageRequest {
+            observeConversationList()
```

Comment:
So we observe the entire conversation list and pick out the right conversation. Any reason why we don't fetch the one-to-one conversation directly?

Reply:
discussed on PM that we will optimize it later on